### ISSUE TITLE
Automatic task discovery

### ISSUE THREAD

**Issue by ac130kz** [GDay/django-q2#30]
From the docs it seems like it's not possible for a cluster to automatically discover tasks directly from code on server boot (e.g. django-dramatiq actors). It is either manually calling the `schedule` function or creating a `Schedule` object.

Is there any way to automatically discover tasks on Django/cluster boot without adding extra creation + cleanup code on Django startup?

**Comment by GDay** [comment]
You are correct, there is currently no way of it discovering tasks. However, I am planning to add that to this library relatively soon.

**Comment by diegodirob** [comment]
Hi guys I'm new to Django-Q2 👋🏻. Coming from Kronos I decided to create a simple decorator to emulate his behavior.
I write the function, use the decorator with Django-Q2 parameters (I have to pass the ID) and he thinks about the creation of the object.
PS. I must necessarily import the file containing the Schedule functions in the app init file.

**Comment by joshuadavidthomas** [comment]
👋 Hello all! I created a new library to address this missing functionality: https://github.com/westerveltco/django-q-registry. 

It's based on something I've been using internal for a while and it's served me fairly well. Has support for defining tasks within a `tasks.py` in any Django app, either via a `register_task` decorator or calling the registry's `register` method directly. All you do is when deploying add an extra call to the `setup_periodic_tasks` management command, and it should all be taken care of for you.

I'd be willing to help out integrating this functionality in to this library, if the maintainers are open to it.

**Comment by GDay** [comment]
Hey @joshuadavidthomas. I would definitely be open to that. Your library looks great, far beyond of what I had in mind! Lovely.

It's definitely a missing feature of Django-q(2). 

Just had a quick look at the source code and in general, it looks really good. Also well done on the test coverage. One thing that I noticed is that you created the model called "Task". That's not going to work well with django-q2 as there is already a model call `Task` to keep track of ran tasks: https://github.com/django-q2/django-q2/blob/88430c90147c07d48dd842759e4992593cf706d4/django_q/models.py#L28.
